src/main/java/mate/academy/springbootintro/dto/cartitem/UpdateCartItemRequestDto.java

```diff
@@ -0,0 +1,9 @@
+package mate.academy.springbootintro.dto.cartitem;
+
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Positive;
+
+public record UpdateCartItemRequestDto(
+        @NotNull @Positive int quantity
```

Comment:
```suggestion
          @Positive int quantity
```
primitive can't be null